creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -1,6 +1,7 @@
 const axios = require('axios')
```

Comment:
the diff is hard to follow but the changes are effectively:
- move most of the logic from the main function to `handleIssueSyncRequest()`, and use the outer function as a wrapper for aggregation
- move all non-core logic into the outer function, e.g. metrics, logging, checking for duplicate sync, and returning response
- change metric from just monitoring `additionalSyncIsRequired()` to monitor full `issueSyncRequest` logic, with a `result` label value for every scenario, including `syncPrimaryFromSecondary()`
- remove legacy logging